database/src/main/scala/com/rasterfoundry/granary/AWSBatch.scala

```diff
@@ -62,9 +62,8 @@ object AWSBatch {
     } map {
       case Left(e)         => Left(e)
       case Right(Left(e))  => Left(e)
-      case Right(Right(r)) =>
-        // trick to suppress "pure expression in statement position" warning
-        Right(locally(r))
+      case Right(Right(_)) =>
```

Comment:
There was a warning for discarding a non-unit value -- as far as I could tell we weren't doing anything with the result and the function signature was `unit` so this seems safe.